data/json/items/generic.json

```
@@ -2793,6 +2793,93 @@
     "weight": "774 g",
     "volume": "1750 ml",
     "flags": [ "SLEEP_AID" ],
+    "variant_type": "generic",
+    "variants": [
+      {
+        "id": "bodypillow_generic",
+        "name": { "str": "anime body pillow" },
+        "description": "A big, body-sized pillow with a print of an anime character on the front and their scantily clad version on the back.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_azurlane",
+        "name": { "str": "ship body pillow" },
+        "description": "A big, body-sized pillow with a print of an aircraft carrier drawn on both sides.  The one on the back is stripped of its armor plating, how naughty.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_gfl",
+        "name": { "str": "gun body pillow" },
+        "description": "A big, body-sized pillow with a print of a fully-modded tacticool M4 SOPMOD II on both sides.  The rifle firing a grenade somehow looks suggestive.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_boston",
+        "name": { "str": "Boston-Chan body pillow" },
+        "description": "A big, body-sized pillow with a print of a blue-haired Boston-Chan, the mascot of Boston AnimeCon on both sides.  This seems to be official merch, and the art on both sides is quite tame.",
```

```suggestion
        "description": "A big, body-sized pillow with a print of a blue-haired Boston-Chan, the mascot of Boston AnimeCon, on both sides.  This seems to be official merch, and the art on both sides is quite tame.",
```
I think you need another comma ther, not sure though

---

I'm unsure either way, actually, so I'll ask my editor friend to double check.

---

editor friend says u right